src/api/doAttachApp.js

```diff
@@ -149,7 +156,19 @@ export const doAttachApp = async (
         }
 
         await Promise.all(promisesSecondRound);
+
+        if (applicationDataOut) {
+            const timeout = timeoutedApps(appTypes).includes(applicationDataOut.application_type_id) ? 10000 : 0;
+            return await checkAppAvailability(applicationDataOut.id, timeout);
+        }
+
+        return {};
     } catch (error) {
+        console.error(error);
```

Comment:
is this console intentional?

Reply:
We are not showing error anywhere in the UI, so I think we should show the error at least in the log.